tests/test-cases/Group1-Docker-Commands/1-4-Docker-Create.robot

```diff
@@ -33,6 +33,19 @@ Create with volume
     #Should Be Equal As Integers  ${rc}  0
     #Should Not Contain  ${output}  Error
 
+Create with named volume
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create -v test-named-vol:/testdir busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+    # TODO: this can be changed to `volume ls -f name=test-named-vol` once filtering by name is supported
```

Comment:
instead of a TODO here, preference would be for a github check so that we get notified to update when filtering issue is closed.